Source/SwiftLintFramework/Extensions/Array+SwiftLint.swift

```diff
@@ -24,3 +24,27 @@ extension Array where Element: NSTextCheckingResult {
         return map { $0.range }
     }
 }
+
+extension Array where Element: Equatable {
+    var unique: [Element] {
+        var uniqueValues: [Element] = []
+        forEach { item in
+            if !uniqueValues.contains(item) {
+                uniqueValues += [item]
+            }
+        }
+        return uniqueValues
+    }
+}
+
+public extension SequenceType {
```

Comment:
`internal extension`? This doesn't seem to be relevant in SwiftLint's public API.